v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/SchemaRegistryOptions.java

```diff
@@ -27,15 +27,15 @@ public interface SchemaRegistryOptions extends PipelineOptions {
       groupName = "Source",
       enumOptions = {
         @TemplateParameter.TemplateEnumOption(
-            KafkaTemplateParamters.MessageFormatConstants.AVRO_CONFLUENT_WIRE_FORMAT),
+            KafkaTemplateParameters.MessageFormatConstants.AVRO_CONFLUENT_WIRE_FORMAT),
         @TemplateParameter.TemplateEnumOption(
-            KafkaTemplateParamters.MessageFormatConstants.AVRO_BINARY_ENCODING),
-        @TemplateParameter.TemplateEnumOption(KafkaTemplateParamters.MessageFormatConstants.JSON)
+            KafkaTemplateParameters.MessageFormatConstants.AVRO_BINARY_ENCODING),
+        @TemplateParameter.TemplateEnumOption(KafkaTemplateParameters.MessageFormatConstants.JSON)
       },
       description = "Kafka Message Format",
       helpText =
           "The format of the Kafka messages to read. The supported values are AVRO_CONFLUENT_WIRE_FORMAT (Confluent Schema Registry encoded Avro), AVRO_BINARY_ENCODING (Plain binary Avro), and JSON.")
-  @Default.String(KafkaTemplateParamters.MessageFormatConstants.AVRO_CONFLUENT_WIRE_FORMAT)
+  @Default.String(KafkaTemplateParameters.MessageFormatConstants.AVRO_CONFLUENT_WIRE_FORMAT)
```

Comment:
nit: Maybe import `MessageFormatConstants` and use `MessageFormatConstants.AVRO_CONFLUENT_WIRE_FORMAT` directly instead of `KafkaTemplateParameters.MessageFormatConstants.AVRO_CONFLUENT_WIRE_FORMAT`? The latter is hard to read for no good reason.

Same for SchemaFormat, etc.

Reply:
Done